conf_files/pyro_camera.yaml

```diff
@@ -0,0 +1,99 @@
+# Config file for a distributed camera & focuser.
+#
+# Contains a default configuration with a simulated camera and focuser for testing
+# and example configurations for other camera and focusers (commented out).
+#
+# Do not edit this file, instead copy one of the examples to pyto_camera_local.yaml
+# before editing as required.
+#
+# Simulated camera and focusers:
+name: camera.simulator.001
+host: 127.0.0.1
```

Comment:
localhost might be better, as this works with IPv6 as well as IPv4... assuming the underlying library can handle this.

Reply:
localhost should work. I'll try it.

Reply:
OK, localhost does work. Now using it in the test configuration. Of course localhost (or its IPv4 or IPv6 equivalents) is only relevant for tests, passing it to either the name server or the camera server prevents any connections between machines.